kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -2748,6 +2748,33 @@ def remove_vm_state_from_cache(uuid):
     if uuid in vm_state_cache:
         del vm_state_cache[uuid]
 
+last_libvirtd_pid_for_vm_priority_sync = None
+def sync_vm_live_priority_after_libvirtd_restart(states):
+    global last_libvirtd_pid_for_vm_priority_sync
+
+    try:
+        current_pid = linux.get_libvirtd_pid()
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)